sfklString.cpp

```diff
@@ -18,14 +18,68 @@
 // along with sfArkLib.  If not, see <http://www.gnu.org/licenses/>.
 
 #include <string.h>
+#include <stdio.h>
+#include "wcc.h"
 
-const char *GetFileExt(const char *FileName)
+char *GetFileExt(char *FileName)

```

Comment:
can't `FileName` still be `const`?

Reply:
It can't be const because we'll use it to edit the string.